test/feature-flags.ts

```diff
@@ -67,7 +81,7 @@ describe('feature flags', function() {
             result[flag] = Math.round((distr[flag] / total) * 100)
         }
         // close enough
```

Comment:
😰

Reply:
it's spot on now when i'm testing 50% of the names instead of sampling every 50th ;)